documentation/docs/mock-apps/services/process-association-event.md

```diff
@@ -0,0 +1,88 @@
+---
+sidebar_position: 26
```

Comment:
I think this `sidebar_position: 26` might overlap with the `Process Aggregation Event` and `Process Object Event` sidebar positions. Could you double check it?

Reply:
Good catch @huynguyen-hl.